packages/dd-trace/src/telemetry/send-data.js

```diff
@@ -1,4 +1,21 @@
 const request = require('../exporters/common/request')
+
+function getHeaders (reqType, debug, application) {
+  const headers = {
+    'content-type': 'application/json',
+    'dd-telemetry-api-version': 'v1',
+    'dd-telemetry-request-type': reqType
+  }
+  if (debug) {
+    headers['dd-telemetry-debug-enabled'] = 'true'
+  }
+  if (application) {
```

Comment:
How could this be missing? These are 2 process-level information that should always be available.

Reply:
I guess they are added by the agent, so there is no need to add them in the tracer.

Reply:
I'm not sure :S, let me check it again.

Reply:
No, they are not added by the agent.

And they are mandatory headers according to the spec

I know the application data is included among the request data but it seems that the headers help to identify the request even if the backend fails to parse the request payload. For example to identify a particular library version and language, sending requests that cannot be parsed.

So it is useful for the backend that the tracer includes them.

Reply:
It's fine if they're needed, but in which case would the condition be false? There would always be an `application`, and this it seems that this would always be true, so why the `if`?

Reply:
Because `sendData` method doesn't guarantee that `application` will always have a value and `getHeaders` method accesses some `application` properties. 
I felt safer with the `if`.
Removed